inspirehep/modules/records/api.py

```diff
@@ -605,8 +605,9 @@ def _query_citing_records(self):
         ref = self._get_ref()
         if not ref:
             raise Exception("There is no $ref for this object")
-        citations = RecordMetadata.query.with_entities(RecordMetadata.id).filter(
-            referenced_records(RecordMetadata.json).contains([ref]))
+        citations = RecordMetadata.query.with_entities(RecordMetadata.id,
+                                                       RecordMetadata.json['control_number']).filter(
+            referenced_records(RecordMetadata.json).contains([ref])).distinct(RecordMetadata.json['control_number'])
```

Comment:
This indentation looks a bit weird...you might want to have something more like:

```python
citation = RecordMetadata.query.with_entities(
    RecordMetadata.id, RecordMetadata.json['control_number']
).filter(referenced_records(RecordMetadata.json).contains([ref]))
.distinct(RecordMetadata.json['control_number'])
```